Code/src/web/views/bolo-list.jade

```diff
@@ -5,7 +5,7 @@ block jumbo
     .jumbotron
         .container
             div#wrapper.row
-                div.col-xs-2.col-md-10
+                div.col-xs-2.col-md-8
```

Comment:
confirm this doesnt mess up the mobile/tablet views, apart from the standard desktop view.